one_fm/patches.txt

```diff
@@ -76,3 +76,4 @@ one_fm.patches.v15_0.update_purchase_order
 one_fm.patches.v15_0.patch_employee_educational_qualifications
 one_fm.patches.v15_0.missing_attendance_request
 one_fm.patches.v15_0.delete_employment_type_field_from_job_opening
+one_fm.patches.v15_0.update_shift_permission_workflow
```

Comment:
This patch might give an error during the merge since the workflow was introduced in this PR. Watch Out!